tests/kafkatest/tests/streams/streams_standby_replica_test.py

```diff
@@ -47,19 +47,23 @@ def __init__(self, test_context):
                                                  })
 
     @cluster(num_nodes=10)
-    @matrix(metadata_quorum=[quorum.isolated_kraft])
-    def test_standby_tasks_rebalance(self, metadata_quorum):
+    @matrix(metadata_quorum=[quorum.combined_kraft],
+            group_protocol=["classic", "streams"])
+    def test_standby_tasks_rebalance(self, metadata_quorum, group_protocol):
         # TODO KIP-441: consider rewriting the test for HighAvailabilityTaskAssignor
         configs = self.get_configs(
-            group_protocol="classic",
+            group_protocol=group_protocol,
             extra_configs=
-            ",sourceTopic=%s,sinkTopic1=%s,sinkTopic2=%s,internal.task.assignor.class=org.apache.kafka.streams.processor.internals.assignment.LegacyStickyTaskAssignor" % (
+            ",application.id=test_standby_tasks_rebalance,sourceTopic=%s,sinkTopic1=%s,sinkTopic2=%s,internal.task.assignor.class=org.apache.kafka.streams.processor.internals.assignment.LegacyStickyTaskAssignor" % (
```

Comment:
Seems the `application.id` is hard-coded here, and inside `StreamsStandByReplicaTest.java` -- wondering it this could lead to issue (in case we change one, but forget to change the other).

Should we change `StreamsStandByReplicaTest.java` and take it as a parameter from the Python code instead?

Reply:
Or just not hardcode it in StreamsStandByReplicaTest. We hardcode it `extra_configs` in other system tests as well, maybe this is a good pattern

Reply:
Done